APIs/EDIDAPI.raml

```diff
@@ -0,0 +1,121 @@
+    get:
+      description: List all the API endpoints for a single sender.
+      responses:
+        200:
+          description: Returned when success.
+          body:
+            example: !include ../examples/sender-get-200.json
+            type: !include schemas/edidapi-sender.json
+        404:
+          description: Returned when the requested Sender ID does not exist.
+    /media-profiles:
+      get:
+        description: Show the last media profile array put before.
+        responses:
+          200:
+            body:
+              example: !include ../examples/media-profiles-get-200.json
+              type: !include schemas/media_profiles.json
+      put:
+        description: Put desired media profiles.
+        body:
+          example: !include ../examples/media-profiles-get-200.json
+          type: !include schemas/media_profiles.json
+        responses:
+          200:
+            description: Returned when success.
+          400:
+            description: >
+              Returned when the media profiles are rejected due to schema validation failure or if the Device doesn't support the requested mode.
+          423:
+            description: Returned when the media profiles can't be applied temporarily (e.g. due to an existing connection).
+          500:
+            description: >
+              Returned if the media profiles can't be applied due to a reason which is not related to a violation of schema.
+            body:
+              type: !include schemas/error.json
+      delete:
+        description: Delete the last media profile array put before.
+        responses:
+          200:
+            description: Returned when success.
+          500:
+            description: >
+              Returned if the media profiles can't delete due to a reason which is not related to a violation of schema.
+            body:
+              type: !include schemas/error.json
+/receivers:
+  displayName: Receivers
+  get:
+    description: List available receivers. All UUIDs are appended with / to indicate that the ID forms part of the URL for the next layer down.
+    responses:
+        200:
+          body:
+            example: !include ../examples/resourcelist-get-200.json
+            type: !include schemas/resource-list.json
+  /{receiverId}:
+    get:
+      description: List all the API endpoints for a single receiver.
+      responses:
+        200:
+          description: Returned when success.
+          body:
+            example: !include ../examples/receiver-get-200.json
+            type: !include schemas/edidapi-receiver.json
+        404:
+          description: Returned when the requested Receiver ID does not exist.
+    /sinks:
+      get:
+        responses:
+          200:
+            description: List of Sinks connected to the Receiver.
+            body:
+              example: !include ../examples/resourcelist-get-200.json
+              type: !include schemas/resource-list.json
+    /edid:
```

Comment:
Should this be an attribute of the sink? An Audio/Video pair of Receivers would have the same sink

Reply:
I agree with this. EDIDs belong to Sinks, Receiver decide on its own how to work with them and fill up Receiver Caps. The BCP recommends how to convert EDID to Receiver Caps and can guide implementers for making straightforward IP -> HDMI Gateways. So there is no an EDID associated with a Receiver itself.